src/DotNetBoilerplate.Core/Projects/Project.cs

```diff
@@ -1,57 +1,60 @@
-﻿namespace DotNetBoilerplate.Core.Projects
+﻿using DotNetBoilerplate.Core.Organizations.Exceptions;
+
+namespace DotNetBoilerplate.Core.Projects
 {
     public class Project
     {
         private Project() { }
         public Guid Id { get; private set; }
         public string Name { get; private set; }
         public string Description { get; private set; }
-        public string Status { get; private set; }
+        public ProjectStatus Status { get; private set; }
         public Guid OrganizationId { get; private set; }
         public Guid CreatorId { get; private set; }
         public DateTimeOffset CreatedAt { get; private set; }
 
-     /*   public enum ProjectStatus
+        public enum ProjectStatus
         {
             Public,
             Private,
             Archived
-        } */
+        } 
 
         public void UpdateName(string name, bool nameIsUnique)
         {
             if (!nameIsUnique)
             {
-                throw new Exception("Invalid name");
+                throw new ProjectNameIsNotUniqueException();
             }
 
             Name = name;
         }
 
-        public void UpdateStatus(string status)
+        public void UpdateStatus(ProjectStatus status)
         {
-            Status = status;
+            if (status == ProjectStatus.Public) { Status = ProjectStatus.Public; }
```

Comment:
To już było w https://github.com/mpienk102/digital-feedback/pull/13 - i jak patrzę to chyba są te same prki więc tutaj uważajcie żeby sobie nie zrobić bałaganu w branchach